MD/md-6/README.md

```diff
@@ -0,0 +1,80 @@
+# MD-6: Suzuka Block Validation
+- **Description**: Provide means to validate blocks to prevent public DA DOS attacks.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+
+
+## Overview
+The current implementation of the Suzuka Network relies on a public and permissionless DA. The blocks are further not validated besides their correctness w.r.t. the Aptos executor. 
+
+This could lead to an unsophisticated DOS attack wherein the attacker simply spams the network with low gas fee--both on the DA and in the execution layer--blocks. A slightly more sophisticated attack would see the attack DOS the DA namespace itself in a [Woods Attack](https://forum.celestia.org/t/woods-attack-on-celestia/59).
+
+While we cannot prevent latter, providing solutions to the former may prove vital to fairness amongst genuine network participants.
+
+## Desiderata
+
+<!--
+  List out the specific desiderata. Each entry should consist of:
+
+  1. Title: A concise name for the desideratum.
+  2. User Journey: A one or two-sentence statement focusing on the "user" (could be a human, machine, software, etc.) and their interaction or experience.
+  3. Description (optional): A more detailed explanation if needed.
+  4. Justification: The reasoning behind the desideratum. Why is it necessary or desired?
+  5. Recommendations (optional): Suggestions or guidance related to the desideratum.
+
+  Format as:
+
+  ### Desideratum Title
+
+  **User Journey**: [user] can [action].
+
+  **Description**: <More detailed explanation if needed (optional)>
+
+  **Justification**: <Why this is a significant or required desideratum>
+
+  **Recommendations**: <Any specific guidance or suggestions (optional)>
```

Comment:
recommendations makes it somewhat unclear what to put here. i would call this `Guidance and suggestions`:

Reply:
yes can take care of this.